app/build.gradle

```diff
@@ -6,37 +6,60 @@ android {
 
     defaultConfig {
         applicationId "cz.muni.fi.pv256.movio2.fk410022"
+        manifestPlaceholders = [applicationLabel: "@string/app_name"]
         minSdkVersion 16
         targetSdkVersion 24
         versionCode 1
         versionName "1.0"
         jackOptions {
             enabled true
         }
+        testInstrumentationRunner "cz.muni.fi.pv256.movio2.fk410022.custom.TestAppRunner"
     }
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true // proguard
+            shrinkResources true
```

Comment:
mam pocit ze toto spomaluje build appky. iba tak vyslovujem pozorovanie, nepouzivala som to este...